components/zerotier/zerotier.app.mjs

```diff
@@ -32,6 +32,48 @@ export default {
         }));
       },
     },
+    memberHidden: {
+      type: "boolean",
+      label: "Member Hidden?",
+      description: "Whether or not the member is hidden in the UI",
+      optional: true,
+      default: null,
+    },
+    memberName: {
+      type: "string",
+      label: "Member Name",
+      description: "User defined name of the member",
+      optional: true,
+      default: null,
+    },
+    memberDescription: {
+      type: "string",
+      label: "Member Description",
+      description: "User defined description of the member",
+      optional: true,
+      default: null,
```

Comment:
Since `memberDescription` is a string, the default value, if included, would need to be a string. I think it's probably best not to include a default value here.
```suggestion
```